com.unity.multiplayer.mlapi/Runtime/Core/NetworkedBehaviour.cs

```diff
@@ -712,15 +719,16 @@ private bool CouldHaveDirtyNetworkedVars()
             return false;
         }
 
-        internal static void HandleNetworkedVarDeltas(List<INetworkedVar> networkedVarList, Stream stream, ulong clientId, NetworkedBehaviour logInstance)
+        internal static void HandleNetworkedVarDeltas(NetworkingManager networkingManager,
+            List<INetworkedVar> networkedVarList, Stream stream, ulong clientId, NetworkedBehaviour logInstance)
         {
-            using (PooledBitReader reader = PooledBitReader.Get(stream))
+            using (var reader = networkingManager.PooledBitReaders.GetReader(stream))
```

Comment:
I get the singleton-side change. But, out of curiosity, what's the rationale for using `var` instead of the type ?

Reply:
It's kind of like the 'auto' specifier in C++.